### PR TITLE
add fields to needs export

### PR DESCRIPTION
**Pull request by rik86future**
Added the Covid19 question, amended the csv export and added a e2e test

image

image

image

### CODE REVIEW

**Comment by rik86future** [comment]
[Add fields to needs export](https://trello.com/c/L7M9eGVU/304-add-fields-to-needs-export)

**Comment by rik86future** [comment]
[Add fields to contact model](https://trello.com/c/UoGJRmZv/303-add-fields-to-contact-model)

File: app/views/shared/_profile-accordion.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <section class="accordion">
 
-    <button class="accordion__control" aria-expanded="true">
+    <button class="accordion__control" aria-expanded="false">
```
Comment (timwis) [inline app/views/shared/_profile-accordion.html.erb]: Why did you change this? This block is expanded by default, so I think this should remain `true`, no?

File: db/schema.rb [hunk 2]
```diff
@@ -58,6 +58,7 @@
     t.jsonb "healthintent_import_data"
     t.string "nhs_number"
     t.date "date_of_birth"
+    t.boolean "has_covid"
```
Comment (timwis) [inline db/schema.rb:61]: Can we change the field name to `has_covid_symptoms`? I think there may be a big difference and easily misinterpreted.

File: app/views/contacts/_extra_questions.html.erb [hunk 3]
```diff
@@ -21,4 +21,15 @@
     <%= form.radio_button :any_children_below_15, false, :id => "any_children_below_15_false" %>
     <label for="any_children_below_15_false" class="radio__label">No</label>
   </div>
+
+  <legend><h2 class="field-section__title">Are you or anyone in your home showing any symptoms of COVID-19?</h2></legend>
```
Comment (timwis) [inline app/views/contacts/_extra_questions.html.erb:25]: These 3 questions should actually be wrapped in their own <fieldset> tags. that's why the display is a bit funky. Not a huge deal though.

File: app/views/shared/_profile-accordion.html.erb (hunk 1 above)
Comment (eddieleeper-nb) [reply app/views/shared/_profile-accordion.html.erb]: I agree - this shouldn't change.
